src/libcore/default.rs

```diff
@@ -78,7 +78,7 @@
 
 #![stable(feature = "rust1", since = "1.0.0")]
 
-/// A trait that types which have a useful default value should implement.
+/// A trait that must be implememnted by types that have a useful default value.
```

Comment:
Why “must”? We do not enforce this invariant, nor will anything break due to breaking this now-a-requirement.